src/components/app-bar/BurgerMenu.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+	import {
+		activeAccount,
+		activeAccountBounties,
+		walletConnect as wcConnection
+	} from '../../stores';
+	import { onMount } from 'svelte';
+
+	let burgerMenuOpen = false;
+	let burgerMenuContainer: HTMLDivElement | null = null;
+
+	function toggleBurgerMenu() {
+		burgerMenuOpen = !burgerMenuOpen;
+
+		if (burgerMenuOpen) {
+			window.addEventListener('click', closeMenuOnOutsideClick);
+		} else {
+			window.removeEventListener('click', closeMenuOnOutsideClick);
+		}
```

Comment:
```suggestion
		$: { if (burgerMenuOpen) {
			window.addEventListener('click', closeMenuOnOutsideClick);
		} else {
			window.removeEventListener('click', closeMenuOnOutsideClick);
		} }
```
If we make it a reactive thing, toggleBurgerMenu and closeBurgerMenu become simpler